src/styles/calendar.scss

```diff
@@ -0,0 +1,27 @@
+.calendar {
+  margin-top: 10%;
+  margin-left: 30%;
```

Comment:
You need to display a calendar in the middle of the screen
<img width="734" alt="Screenshot 2022-07-21 at 21 31 25" src="https://user-images.githubusercontent.com/47606379/180287629-9ddb5cf9-3348-4b71-b9be-45f32ba4dc2b.png">